src/App.js

```diff
@@ -1,15 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+// import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
 
+//<ChatEntry
+          //sender={chatMessages[0].sender}
+          //body={chatMessages[0].body}
+          //timeStamp={chatMessages[0].timeStamp}
+        ///>
 const App = () => {
+  const[likesCount,setLikeCount]=useState(0);
+  const updateLikes=(isLike)=>{
+    if (isLike){
+      setLikeCount(likesCount+1);
+    } else {
+        setLikeCount(likesCount-1)
+
+    }
+}
```

Comment:
This is a good start. However, your program is only changing the state of the likesCount and doesn't ever address the chatMessages. When you change the buttons, the likes will change visually, but each message's backend data will remain unchanged.